packages/components/card/card.scss

```diff
@@ -54,20 +53,18 @@ $card-border-hover-color: $color_nhsuk-grey-3;
 .nhsuk-card--clickable {
   border-bottom-width: $card-border-bottom-width;
 
-  &:hover,
-  &:active {
-    cursor: pointer;
-
-    .nhsuk-card__heading a,
-    .nhsuk-card__link {
-      color: $nhsuk-link-hover-color;
-      text-decoration: none;
-
-      &:focus {
-        color: $nhsuk-focus-text-color;
-      }
+  .nhsuk-card__heading a,
+  .nhsuk-card__link {
+    &::before {
+      background-color: rgba(255, 255, 255, 0); /* [8] */
```

Comment:
@chrimesdev Don't know whether we want to carry IE10 baggage or not? It's a relatively innocuous little hack to force IE10 and 9 to render the clickable overlay. But we don't "officially support" IE10.

Your call...

Reply:
Happy to include this - 1 line of code to make it work in a whole browser isn't bad!